php-practice/php-practice_1.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+// Q1 変数と文字列
+$name = "細川";
+echo "私の名前は「" . $name . "」です。" . PHP_EOL;
+
+// Q2 四則演算
+$num = 5 * 4;
+echo $num . PHP_EOL;
+echo ($num / 2) . PHP_EOL;
+
+// Q3 日付操作
+date_default_timezone_set('Asia/Tokyo');
+$date_time = date("Y年m月d日 H時i分s秒");
+echo "現在時刻は、" . $date_time . "です。" . PHP_EOL;
+
+// Q4 条件分岐-1 if文
+$device = "windows";
+if ($device === "windows"||$device==="mac") {
```

Comment:
[nits]
細かいですが、空白や改行は統一しましょう。
揃えることで可読性が高くなります！
```suggestion
if ($device === "windows"|| $device==="mac") {
```

Reply:
https://github.com/minnhei186/php-practice/pull/1/commits/18981acc9db536a640f5990f963666bc5ff80db8

修正を行いました。
以後気をつけます。